app/models/AuthUserSchema/AuthUserSchema.js

```diff
@@ -0,0 +1,44 @@
+import mongoose from "mongoose";
+import bcrypt from "bcrypt";
```

Comment:
Going through the comment in which I said that schema middleware should be used in endpoints, there is no reason and sense in importing and using `bcrypt` here